subsys/bluetooth/host/Kconfig

```diff
@@ -646,6 +646,13 @@ config BT_DEBUG_SDP
 		This option enables debug support for the Bluetooth
 		Service Discovery Protocol (SDP).
 
+config BT_DEBUG_DIS
+	bool "Bluetooth Device Information Service(DIS) debug"
+	depends on BT_GATT_DIS
+	help
+		This option enables debug support for the Bluetooth
+		Device Information Service(DIS).
+
```

Comment:
Id split this change in a separate patch.

Reply:
I can do that

Reply:
I spoke to Johan about this change, we agreed that it better to have just one option which enables debug for every service as oppose to have one per service where we may end up bloating the system with debug options. Also the help indentation is wrong, it seems you have copied it from SDP which is also wrong (it should be just 2 spaces.)

Reply:
Agree

Reply:
Changed